ingest-mongodb-public/environments/production.yml

```diff
@@ -0,0 +1,36 @@
+image:
+  repository: node
+  tag: 18
+
+cronJobs:
+  - name: ingest-all-content
+    schedule: "0 6 * * *" # every day at 6am UTC (1AM EST)
+    command: ["node", ".", "all"]
```

Comment:
Will this still work given that this is no longer in `ingest`? Do we need to update/move the `ingest-service.dockerfile` to this project?

Reply:
Yep, good catch. Need to do this. Will do next

Reply:
ticket for this pls

Reply:
No? It's done as part of this PR